splitgraph/commandline/cloud.py

```diff
@@ -1310,6 +1310,29 @@ def tunnel_c(remote: str, local_address: str, namespace_and_repository: Optional
         start_repository_tunnel(remote, local_address, namespace_and_repository)
 
 
+@click.command("sync-to-seafowl")
+@click.option("--remote", default="data.splitgraph.com", help="Name of the remote registry to use.")
+@click.argument("vdb_id", type=str)
+@click.argument("seafowl_hostname", type=str)
+@click.argument("seafowl_secret", type=str)
```

Comment:
Could use a password prompt here if the users don't want the secret to end up in their shell history (https://click.palletsprojects.com/en/7.x/options/#password-prompts)